src/Common/tests/System/IO/Compression/LocalMemoryStream.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
While these changes aren't wrong can you help me understand why all these clean-up type changes are included here?